code/game/verbs/ooc.dm

```diff
@@ -38,6 +38,14 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 	if(!msg)
 		return
 
+	var/lower_msg = lowertext(msg)
+	var/list/bad_tastes = list("i like mercedes-amg", "i like the toyota prius", "i want a g-wiz", "automatic cars are fine")
```

Comment:
I am open to suggestions on other things to add

Reply:
Please add something related to the fucking bikers that like to swerve in front of you going 20 in a 40 instead of staying in their fucking bike lane

Reply:
> Please add something related to the fucking bikers that like to swerve in front of you going 20 in a 40 instead of staying in their fucking bike lane

OH SHIT CYCLISTS